.github/workflows/release.yml

```diff
@@ -108,7 +108,8 @@ jobs:
             --form "file=@${asana_dsyms_path};type=application/zip"
         fi
 
-    - name: Send Mattermost message
+      - name: Send Mattermost message
+      if: ${{ success() || failure() }} # Don't execute when cancelled
```

Comment:
perhaps we should just check: if: ! ${{ cancelled() }}

Reply:
We could, but on the other hand if any other status shows up in the future, json won’t have mapping for it. Existing approach will allow only for those two explicit statuses to be processed. I’ll keep it how it is if you don’t mind.